support/gathering-cluster-data.adoc

```diff
@@ -5,5 +5,16 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
+When opening a support case, it is often helpful to provide debugging
+information about your cluster to Red Hat Support.
+
+It is recommended to provide:
+
+* xref:../support/gathering-cluster-data.adoc#support_gathering_data_gathering-cluster-data[Data gathered using the `oc adm must-gather` command]
```

Comment:
This looks like an xref to a module? If so, then it will need revision. If not (i.e., if it to an assembly), then should be fine. Same with next xref.

Reply:
@bobfuru You're allowed to put xrefs to anything (assemblies or modules) as long as you're in an assembly. You can NOT put an xref to anything if you're in a module.